src/irmin-graphql/server.ml

```diff
@@ -335,7 +335,7 @@ struct
                           | `Node ->
                               Store.Tree.get_tree tree relative_key >|= fun t ->
                               Lazy.(force tree_as_node (t, absolute_key)))
-                  >>= Lwt.return_ok);
+                  >>= fun r -> Lwt.return (Ok r));
```

Comment:
is there an ocamlformat option to always put these statements on their own lines?

Also, I'd recommend using `>|= fun r -> Ok r` instead of introducing an intermediate closure.

Reply:
> is there an ocamlformat option to always put these statements on their own lines?

I think the closest option to this is `break-infix = fit-or-vertical` which is already activated. It will it these on their own lines if the whole expression does not fit in one line.

> Also, I'd recommend using `>|= fun r -> Ok r` instead of introducing an intermediate closure.

Indeed. There are other places where we could do that. I'll review them.